scrapper/controllers/__init__.py

```diff
@@ -4,21 +4,17 @@
 from flask import render_template, request, redirect, Blueprint, abort
 from flask.ext.api import status
 
+from scrapper.controllers.twitter import twitter_bp
 from scrapper.controllers.miner import mine_user
 
 
-twitter = Blueprint('twitter', __name__, url_prefix = '/twitter')
-
-@twitter.route('/', methods=['GET', 'POST'])
+@twitter_bp.route('/', methods=['GET', 'POST'])
 def index():
     abort(400)
 
-@twitter.route('/<user_name>')
+@twitter_bp.route('/<user_name>')
 def show_user(user_name=None):
     refresh = request.args.get('refresh', '')
-    user = miner.mine_user(user_name, refresh)
-    user = json.dumps(user)
+    user = miner.mine_user.delay(user_name, refresh)
```

Comment:
Exactly. Remember workers are asynchronous. The scraper might only be triggered 5 minutes after the call, and you cannot keep the request waiting for that long.

In real life, stuff are not done like this. There's usually one endpoint for async creation of resources and another one to get them. Or even a callback with some push action when the resource is finally created.

But I see no point in getting this exercise to this level. The main point here is make you understand HTTP return codes and that they are important for clients consuming your API, besides getting you to begin understanding how celery works.